src/analysis/convex/star.lean

```diff
@@ -0,0 +1,453 @@
+/-
+Copyright (c) 2021 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import analysis.convex.basic
+
+/-!
+# Star-convex sets
```

Comment:
It might be worth a note in this docstring somewhere that this has nothing to do with `has_star`.